### ISSUE TITLE
Model download failure

### ISSUE THREAD

**Issue by Haoxuanli-Thu** [MIAGroupUT/carotid-segmentation#1]
I want to download model from the website https://surfdrive.surf.nl/files/index.php/s/evopHTobixCuf3t/download, but the site shows "Unspecified share exception"

 By the command "make get-models,This error will appear:
 Archive: contour_transform_dropout.tar End-of-central-directory signature not found.  Either this file is not a zipfile, or it constitutes one disk of a multi-part archive.  In the latter case the central directory and zipfile comment will be found on the last disk(s) of this archive. unzip:  cannot find zipfile directory in one of contour_transform_dropout.tar orcontour_transform_dropout.tar.zip, and cannot find contour_transform_dropout.tar.ZIP, period.

**Comment by 14thibea** [comment]
Hello,

Thank you for your interest in the project.
We moved the data without updating the paths of the repo. I updated all the paths now in the README.md and the sh files that are called by `make`.
The models are available at: [https://surfdrive.surf.nl/files/index.php/s/ywbP34RlUyuMIZU](https://surfdrive.surf.nl/files/index.php/s/ywbP34RlUyuMIZU).

With kind regards,

**Comment by Haoxuanli-Thu** [comment]
thanks，But I have come across a new problem, I hope you can answer it。
I retrained the CNN network by using the contour training instructions, adopted the default 5-fold cross-validation, and iterated almost 200 times. The training loss and validation loss were shown in the figure. After visualization, the prediction results were not ideal, and even the contour points were outside the figure
![loss](https://github.com/MIAGroupUT/carotid-segmentation/assets/92614624/e449a749-299a-4202-8962-450ea0b8a6b3)
